src/main/java/com/zextras/utils/ReflectionUtils.java

```diff
@@ -0,0 +1,52 @@
+package com.zextras.utils;
+
+import com.zextras.transcoders.reflection.DefaultReflectionTranscoder;
+
+import java.lang.reflect.Field;
+
+/**
+ * Helper util class that performs reflecting operations.
+ */
+public class ReflectionUtils {
```

Comment:
Utility classes should not have public or default constructor.

Reply:
i didnt get the point about the constructor
there are no constructors

Reply:
Please check [this](https://github.com/Zextras/slcw/pull/2#discussion_r985626361) comment.

Reply:
got it!